public/components/game/details/details_test.js

```diff
@@ -13,6 +13,7 @@ var DetailsViewModel = details.ViewModel;
 QUnit.module("bitballs/game/details/", {
     setup: function() {
         localStorage.clear();
+        fixture.delay = 1;
```

Comment:
if this isn't here, do tests break? If so, we need to get to the bottom of this.  Can you create an issue for this investigation?